### ISSUE TITLE
ASTER_GDEMV3 does not deliver elevation data

### ISSUE THREAD

**Issue by ChristophKubik** [dem-net/DEM.Net#104]
**Describe the bug**
When trying to get elevation data from the ASTER_GDEMV3 dataset, there is always null returned. As it turns out, when trying see if some of the DEM files are missing, the `ElevationService` cannot find a coverage for the given coordiantes (screenshot). Then no files are downloaded and therefore the elevation service cannot extract the needed data for the given point (second warning in screenshot) and returns null.

**To Reproduce**
Can be reproduced with the ElevationSample.cs from the sample repository.
In my project I use DEMNet the get data from a geometry line but in the sample repo it can be reproduced with both, the single point elevation and the geometry line.

**Expected behavior**
The missing files should be downloaded and then a valid GeoPoint should be returned.

**Screenshots**
![image](https://user-images.githubusercontent.com/17759208/186359138-52055c8f-ba9a-43f1-b024-73f0ad4ef36d.png)

**Desktop (please complete the following information):**
OS: Win10
Fails for .net core 3.1 & .net6
Fails for DEMNet 0.2.8 & 0.3.2

**Comment by xfischer** [comment]
Hi @ChristophKubik, sorry for the late reply, I'll have a look.
I found a problem with georeferencing in the tiles so that may be a problem. Does this work with another DEM ? (SRTM for example)

**Comment by ChristophKubik** [comment]
@xfischer Yes, I have both of the SRTM and the NASADEM in use, they all work and use the exact same code.

**Comment by xfischer** [comment]
Can you try to remove all files from you LocalData\DEM.Net\ASTER_GDEMV3 ? 
That may trigger another download of fresh data.
If it still not working then it's the data source itself that may have changed.

**Comment by xfischer** [comment]
Did you add you nasa credentials to appsettings.json file ?

**Comment by ChristophKubik** [comment]
Yes the login credentials are there and I can login to their website. This already used to work about a year ago.
Deleting the files didn't change anything. When not developing, my application runs in a linux docker container which has no mounted volume and therefore the local data will allways be empty as soon as I redeploy (which currently happens quiet frequent) .
